flight_computer/packets.py

```diff
@@ -70,35 +124,49 @@ def data(self) -> dict:
     @data.setter
     def data(self, data: dict) -> None:
         """Set data, validating it to make sure it specifies exactly the right fields"""
-        for key in data.keys():
-            if key not in self._names:
-                print("Invalid key in data: " + key)
-        for key in self._names:
-            if key not in data:
-                print("Missing key: " + key)
+        for field in data.keys():
+            if field not in self._fields:
+                print("Invalid key in data: " + field)
```

Comment:
These should probably be exceptions, that's my bad since I included this code originally.